en/guide/async-data.md

```diff
@@ -108,4 +110,4 @@ export default {
 }
 ```
 
-To customize the error page, take a look at the [VIEWS layouts section](/guide/views#layouts).
+To customize the error page, take a look at the [Guide Views `layouts`](/guide/views#layouts).
```

Comment:
To reason for all capitalise Word. To keep consistency with others link. I propose to present link in the format « Section Page SectionPage » like others link (e.g. API Pages `asyncData`) or (API Essential `context`)